.pre-commit-config.yaml

```
@@ -3,6 +3,7 @@ repos:
     rev: v2.3.0
     hooks:
     -   id: check-yaml
+        args: [--unsafe]
```

Could exclude instead though I think this is the only `yml` in the repo so the `--unsafe` flag seems better 
```suggestion
        exclude: mkdocs\.yml
```